tokio-console/src/state/tasks.rs

```diff
@@ -217,16 +221,35 @@ impl TasksState {
                     warnings: Vec::new(),
                     location,
                 };
-                task.lint(linters);
+                if matches!(task.lint(linters), TaskLintResult::RequiresRecheck) {
+                    next_pending_lint.insert(task.id);
+                }
```

Comment:
nit: It's more idiomatic Rust to use `if let` here.
```suggestion
                if let TaskLintResult::RequiresRecheck = task.lint(linters) {
                    next_pending_lint.insert(task.id);
                }
```